uat/testing-features/src/main/resources/greengrass/components/recipes/client_ipc.yaml

```diff
@@ -9,9 +9,14 @@ ComponentName: aws.greengrass.client.IpcClient
 ComponentVersion: '1.0.0'
 ComponentDescription: IPC Client for tests
 ComponentPublisher: Amazon
+ComponentDependencies:
+  aws.greengrass.clientdevices.mqtt.Bridge:
+    VersionRequirement: ">=1.0.0"
```

Comment:
There is no version 1.0.0 of Bridge component.
Please take a look https://docs.aws.amazon.com/greengrass/v2/developerguide/mqtt-bridge-component.html#mqtt-bridge-component-versions

I guess because it is greengrass v2.